### PR TITLE
docs: clarify markdown vs shortcode use cases for images

### PR DESCRIPTION
**Pull request by imatwawana**
Add clarifying text for when to use markdown vs a shortcode for adding images in docs

### CODE REVIEW

File: docs/sources/writing-guide/shortcodes/index.md [hunk 1]
```diff
@@ -79,7 +79,10 @@ The following shortcode inserts the latest version of `shared-page.md` from the
 
 ## `figure` shortcode
 
-The `figure` shortcode renders an image with a caption using an HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes) element. To add a figure, insert the `figure` shortcode with the following named parameters:
+The `figure` shortcode renders an image with a caption using an HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes) element. This shortcode allows you more control over how an image is rendered, but if you don't need these options, you can use [basic markdown to add images]({{< reflref "../markdown-guide/#images" >}}). 
+
+
```
Comment (imatwawana) [inline docs/sources/writing-guide/shortcodes/index.md:84]:
```suggestion
```

File: docs/sources/writing-guide/shortcodes/index.md [hunk 2]
```diff
@@ -79,7 +79,9 @@ The following shortcode inserts the latest version of `shared-page.md` from the
 
 ## `figure` shortcode
 
-The `figure` shortcode renders an image with a caption using an HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes) element. To add a figure, insert the `figure` shortcode with the following named parameters:
+The `figure` shortcode renders an image with a caption using an HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes) element. This shortcode allows you more control over how an image is rendered, but if you don't need these options, you can use [basic markdown to add images]({{< reflref "../markdown-guide/#images" >}}).
```
Comment (MattDodsonEnglish) [inline docs/sources/writing-guide/shortcodes/index.md]:
```suggestion
The `figure` shortcode renders an image with a caption using an HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes) element. This shortcode allows you more control over how an image is rendered, but if you don't need these options, you can use [basic Markdown to add images]({{< reflref "../markdown-guide/#images" >}}). 
```

Comment (imatwawana) [inline docs/sources/writing-guide/shortcodes/index.md:82]:
```suggestion
The `figure` shortcode renders an image with a caption using an HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes) element. This shortcode allows you more control over how an image is rendered, but if you don't need these options, you can use [basic markdown to add images]({{< relref "../markdown-guide/#images" >}}). 
```

Comment (MattDodsonEnglish) [reply docs/sources/writing-guide/shortcodes/index.md]: It seems Writers' Toolkit conventions thus far capitalize "Markdown", so I say continue the tradition.

**Review by MattDodsonEnglish — APPROVED** [review]
Approved w/ capitalization nit.

File: docs/sources/writing-guide/markdown-guide/index.md [hunk 3]
```diff
@@ -171,7 +171,9 @@ _Do not_ use image shortcodes at this time. Instead, include images in a documen
 
 This follows the format `![alt text](URL)`.
 
-Within Markdown, HTML is valid and to be used sparingly:
+Alternatively, you can use the [figure shortcode]({{< relref "../shortcodes/#figure-shortcode" >}} if you need more image options, such as adding captions or controlling the image size.
```
Comment (imatwawana) [inline docs/sources/writing-guide/markdown-guide/index.md:174]:
```suggestion
Alternatively, you can use the [figure shortcode]({{< relref "../shortcodes/#figure-shortcode" >}}) if you need more image options, such as adding captions or controlling the image size.
```

File: docs/sources/writing-guide/shortcodes/index.md (hunk 2 above)
Comment (imatwawana) [reply docs/sources/writing-guide/shortcodes/index.md]: Thanks for the catch!

**Pull request merged**